app/models/campaign.rb

```diff
@@ -107,6 +107,21 @@ def seller_distributor_pairs
     pairs
   end
 
+  def display_name
```

Comment:
this is a bit of a rendering concern, so i could see a case for having the client pull the project name and not have this on the campaign model. but i think it would also be good to centralize the logic

Reply:
I think this is a good solution